src/components/PDF/ApplicationForm.tsx

```diff
@@ -19,14 +19,14 @@ interface Props {
 }
 
 interface State {
-  formQuestions: any[] | undefined,
+  fields: any[] | undefined,
```

Comment:
In future PR please annotate the fields so we remove any[]s. This is bad practice because any[] circumvents strong typing which is the whole reason we are using typescript

Reply:
Done